### PR TITLE
Provide a new spackage for py-pint.

### PR DESCRIPTION
**Pull request by KineticTheory**
* **Pint** is a Python package to define, operate and manipulate physical quantities: the product of a numerical value and a unit of measurement. It allows arithmetic operations between them and conversions from and to different units. 
* https://pint.readthedocs.io/en/latest/

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pint/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class PyPint(PythonPackage):
+    """Pint is a Python package to define, operate and manipulate physical
+    quantities: the product of a numerical value and a unit of measurement.
+    It allows arithmetic operations between them and conversions from and
+    to different units."""
+
+    homepage = "https://pypi.python.org/pypi/pint"
+    url      = "https://pypi.io/packages/source/p/pint/Pint-0.8.1.tar.gz"
+
+    version('0.8.1', 'e1f80f3f8fc4e61f68ad3912db26b3a8')
```
Comment (healther) [inline var/spack/repos/builtin/packages/py-pint/package.py]: could you use the sha256 hash? You should have gotten that directly when you did `spack create`

**Comment by KineticTheory** [comment]
@healther I updated the has to sha256 as requested.  Sorry about the oversight.  For stability we are using a version of spack from mid-2017 (before the switch to sha256) and I forgot about the switch to sha256 used in `develop`.

File: var/spack/repos/builtin/packages/py-pint/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    quantities: the product of a numerical value and a unit of measurement.
+    It allows arithmetic operations between them and conversions from and
+    to different units."""
+
+    homepage = "https://pypi.python.org/pypi/pint"
+    url      = "https://pypi.io/packages/source/p/pint/Pint-0.8.1.tar.gz"
+
+    version('0.8.1', 'afcf31443a478c32bbac4b00337ee9026a13d0e2ac83d30c79151462513bb0d4')
+
+    depends_on('py-setuptools', type='build')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-pint/package.py]: @KineticTheory I was reading the README bundled with the [sources](https://github.com/pauleve/pint/archive/2018-07-10.zip):

> Pint requires the libRmath standalone library for the stochastic simulation and parameters
inference from time interval specifications.
You can disable this functionnality with the following command:
>
> 	$ python setup.py --disable-R

Are we missing something here (I see neither a dependency on R nor disabling of dependencies)?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pint/package.py]: @alalazo This is for a different `pint` library. For this library, from the [README](https://github.com/hgrecco/pint/blob/master/README):

> - Dependency free: it depends only on Python and its standard library.

Comment (alalazo) [reply var/spack/repos/builtin/packages/py-pint/package.py]: Sorry, didn't check the link, just searched the name.

**Pull request merged**